index.js

```diff
@@ -346,29 +349,31 @@ jsDataAdapter.Adapter.extend({
         var _this5 = this;
 
         return new jsData.utils.Promise(function (resolve, reject) {
-            var dsQuery = _this5.datastore.createQuery(_this5.getKind(mapper, opts));
-            dsQuery = _this5.filterQuery(dsQuery, query, opts);
-            dsQuery = dsQuery.select('__key__');
-            _this5.datastore.runQuery(dsQuery, function (err, entities) {
-                if (err) {
-                    console.error(err);
-                    return reject(err);
-                }
-                var keys = entities.map(function (entity) {
-                    return entity.key;
-                });
-                _this5.datastore.delete(keys, function (err, apiResponse) {
-                    if (err) {
-                        console.error(err);
-                        return reject(err);
-                    }
-                    resolve([undefined, apiResponse]);
-                });
-            });
+						try {
+								var dsQuery = _this5.datastore.createQuery(_this5.getKind(mapper, opts));
```

Comment:
You're using tabs as indentation characters. While historically correct, we use spaces by default to make sure the code shows the same in every editor.